.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,12 @@
+{
+  "name": "nixpkgs",
+  "image": "mcr.microsoft.com/devcontainers/universal:2-linux",
+  "features": {
+    "ghcr.io/devcontainers/features/nix:1": {
+      "multiUser": false
```

Comment:
Auto enable flake support by adding `"extraNixConfig": "experimental-features = nix-command flakes",`

Add `"packages": "nixpkgs-fmt,nixd"` to support nix IDE and formatter.